moda-contracts/src/ModaRegistry.sol

```diff
@@ -107,9 +129,11 @@ contract ModaRegistry is IModaRegistry, AccessControlEnumerable {
      */
     function removeManagers(address[] calldata managers) external {
         for (uint256 i = 0; i < managers.length; i++) {
-            if (managers[i] == address(0)) revert AddressCannotBeZero();
-            _managers[msg.sender].remove(managers[i]);
-            emit ManagerRemoved(msg.sender, managers[i]);
+            if (managers[i] == address(0)) {
```

Comment:
I don't think this matters and it wastes gas

Reply:
Removed.